FordLabs/PeopleMover
# Change landing page text, adjust font size of text

## Issue
Resolves #496 

## What was done
- [x] Adjust font size of general text on landing page
- [x] Remove text from landing page
- [x] Change button text to 'Get Started'

## How to test
1. Go to landing page, double check the button, and the landing page text

Feature URL: https://featurebranchui.apps.pd01i.edc1.cf.ford.com/
Feature Branch Deployment: https://jenkins-fordlabs.apps.pd01e.edc1.cf.ford.com/job/PeopleMover/job/Deploy%20PeopleMover%20FeatureBranch/
